about.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header>
+        <div class="logLine">
+            <ul>
+                <li class="icon"><img src="images/Twitter.png" alt="Twitter"></li>
+                <li class="icon"><img src="images/Github.png" alt="Github"></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log Out</a></li>
+            </ul>
+        </div>
+        <div class="menuIcon">
+            <span class="ban"></span>
+            <span class="ban"></span>
+            <span class="ban"></span>
+        </div>
+        <nav>
+            <a href="index.html" id="logo"><img src="images/logo.png" alt="logo"></a>
+            <ul class="nav">
+                <li><a class="ml" href="about.html">About</a></li>
```

Comment:
- [x] It should be possible to navigate between the pages, kindly consider adding a link from the `About` page to the `Home` page, you can change this link to the Home link
```suggestion
                <li><a class="ml" href="index.html">Home</a></li>
```